gr-qtgui/lib/freq_sink_c_impl.cc

```diff
@@ -74,6 +75,11 @@ namespace gr {
       d_argv = new char;
       d_argv[0] = '\0';
 
+      // setup bw input port
+      message_port_register_in(d_port_bw);
+      set_msg_handler(d_port_bw,
+                      boost::bind(&freq_sink_c_impl::handle_set_bw, this, _1));   
```

Comment:
Ideally, we'd not use `boost::bind` (see also our GREP on coding standards). But it's OK for now.

Reply:
Agreed, but a rewrite didn't seem desirable at this point.